frozendict/FrozenDict.py

```diff
@@ -43,9 +44,26 @@ def __new__(cls, *args, **kwargs):
     def __init__(self,
                  value: Mapping[KT, VT_co] = None,
                  *,
-                 homogeneous_type=False,
-                 remove_none_value=False,
+                 homogeneous_type: bool = False,
+                 remove_none_value: bool = False,
```

Comment:
Maybe change to `remove_none_values` (plural)?
```suggestion
                 remove_none_values: bool = False,
```